owl/SOMA-SAY.owl

```diff
@@ -121,6 +341,74 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#ESTSchemaTheory -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA.owl#ESTSchemaTheory">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#ImageSchemaTheory"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#defines"/>
+                <owl:someValuesFrom rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#PhysicalObjectRole"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#defines"/>
+                <owl:cardinality rdf:datatype="http://www.w3.org/2001/XMLSchema#nonNegativeInteger">1</owl:cardinality>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#string">A schematic theory that describes the existence of an entity, thereby having one Role, i.e. what entity is asserted to exist.
```

Comment:
I am still a bit puzzled about this sentence.
So well, an object exists within its lifetime (which is an event).
It surely participates in this event, but it is not clear to me how and why an the participation of the object would be classified.
The last piece states that this role is "what entity is asserted to exist". How is that a role?

Maybe some illustration would help here? Or more details about the lifetime event and particpation

Reply:
The comment for the role is much more clear, in my opinion